modApiServer/src/org/aion/api/server/http/ApiWeb3Aion.java

```diff
@@ -660,9 +660,14 @@ else if (_params instanceof JSONObject) {
             return new RpcMsg(null, RpcError.INVALID_PARAMS, "Null raw transaction provided.");
 
         byte[] rawTransaction = ByteUtil.hexStringToBytes(_rawTx);
-        byte[] transactionHash = sendTransaction(rawTransaction);
 
-        return new RpcMsg(TypeConverter.toJsonHex(transactionHash));
+        try {
```

Comment:
this try catch is unnecessary; 

Heuristic that I've imposed in the web3 implementation: any exceptional circumstances (ie. internal errors) in the execution of an remote procedure call should be caught by the top-level try catch:

https://github.com/aionnetwork/aion/blob/708dfee890d05f484de082c8fb6a5115d8c5ba38/modApiServer/src/org/aion/api/server/rpc/RpcProcessor.java#L80

UNLESS the exception is caused by something the user can do something about (like invalid address length, account unlocks, etc.) 

Decipline around error handling really important in languages like java, where you could end up polluting your codebase with error handling and make it more fragile and less readable by throwing ad-hoc try catches everywhere. 

@qoire - opinions?